libmamba/src/api/configuration.cpp

```diff
@@ -1776,11 +1786,30 @@ namespace mamba
                                          ctx.prefix_params.root_prefix / "condarc.d",
                                          ctx.prefix_params.root_prefix / ".mambarc" };
 
-        std::vector<fs::u8path> home = { env::home_directory() / ".conda/.condarc",
-                                         env::home_directory() / ".conda/condarc",
-                                         env::home_directory() / ".conda/condarc.d",
-                                         env::home_directory() / ".condarc",
-                                         env::home_directory() / ".mambarc" };
+        std::vector<fs::u8path> conda_user = {
+            env::user_config_dir() / "../conda/.condarc",
+            env::user_config_dir() / "../conda/condarc",
+            env::user_config_dir() / "../conda/condarc.d",
+            env::home_directory() / ".conda/.condarc",
+            env::home_directory() / ".conda/condarc",
+            env::home_directory() / ".conda/condarc.d",
+            env::home_directory() / ".condarc",
+        };
+        if (env::get("CONDARC"))
+        {
+            conda_user.push_back(fs::u8path(env::get("CONDARC").value()));
+        }
+
+        std::vector<fs::u8path> mamba_user = {
+            env::user_config_dir() / ".mambarc",      env::user_config_dir() / "mambarc",
```

Comment:
@pavelzw do you think we should be adding these? I don't have a strong opinion. Nobody will use them anyways.

Reply:
I think actually @0xbe7a uses them 😄 so yes